packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -328,20 +331,29 @@ describe("BnsConnection", () => {
   it("can register a blockchain", async () => {
     pendingWithoutBnsd();
     const connection = await BnsConnection.establish(bnsdTendermintUrl);
-    const chainId = await connection.chainId();
+    const registraChainId = await connection.chainId();
 
     const profile = new UserProfile();
     const wallet = profile.addWallet(Ed25519HdWallet.fromEntropy(await Random.getBytes(32)));
     const identity = await profile.createIdentity(wallet.id, HdPaths.simpleAddress(0));
     const identityAddress = keyToAddress(identity.pubkey);
 
     // Create and send registration
-    const blockchainId = `wonderland_${Math.random()}` as ChainId;
+    const chainId = `wonderland_${Math.random()}` as ChainId;
+    const blockchainId = toUtf8(chainId) as BnsBlockchainId;
     const registration: RegisterBlockchainTx = {
       kind: TransactionKind.RegisterBlockchain,
-      chainId: chainId,
+      chainId: registraChainId,
```

Comment:
Correct, we register on one chain, but the chain being registered is a different id

Reply:
Good. Now I only need to fix the typo in `registraChainId`